tests/test_pipeline/test_pipeline.py

```diff
@@ -439,15 +441,82 @@ def test_get_fold_info_interface_hours(catboost_pipeline: Pipeline, example_tsdf
     assert expected_columns == sorted(info_df.columns)
 
 
+def test_get_fold_info_refit_true(catboost_pipeline: Pipeline, example_tsdf: TSDataset):
+    """Check that Pipeline.backtest returns info dataframe with correct train with regular refit."""
+    _, _, info_df = catboost_pipeline.backtest(
```

Comment:
May be we can use light pipeline with Naive model or sth like this for this test?